src/app/decorators/index.ts

```diff
@@ -0,0 +1 @@
+export * from './resporse/response.decorator';
```

Comment:
rename resporse to **response**
and one more question why this folder has only an index file? fill free to move all decorators here

Reply:
this is a draft structure for the future, when there will be more decorators